helpers/config.py

```diff
@@ -1194,21 +1195,17 @@ def __questions_dev_mode(self):
                     not self.__dict['kc_dev_build_id'] or
                     self.__dict['kc_path'] != kc_path
                 ):
-                    build_id = '{prefix}{timestamp}'.format(
-                        prefix=self.get_prefix('frontend'),
-                        timestamp=str(int(time.time()))
-                    )
-                    self.__dict['kc_dev_build_id'] = build_id
+                    prefix = self.get_prefix('frontend')
+                    timestamp = str(int(time.time()))
+                    self.__dict['kc_dev_build_id'] = f'{prefix}{timestamp}'
 
                 if (
                     not self.__dict['kpi_dev_build_id'] or
                     self.__dict['kpi_path'] != kpi_path
                 ):
-                    build_id = '{prefix}{timestamp}'.format(
-                        prefix=self.get_prefix('frontend'),
-                        timestamp=str(int(time.time()))
-                    )
-                    self.__dict['kpi_dev_build_id'] = build_id
+                    prefix = self.get_prefix('frontend')
+                    timestamp = str(int(time.time()))
```

Comment:
Is it necessary to have the `str(int(time.time()))` everywhere if it's being put inside a formatted string anyway?
```python
>>> f'{str(int(time.time()))}' == f'{int(time.time())}'
True
```